app/src/main/java/org/resolvetosavelives/red/newentry/PatientEntryScreen.kt

```diff
@@ -80,10 +80,6 @@ class PatientEntryScreen(context: Context, attrs: AttributeSet) : RelativeLayout
     }
     TheActivity.component.inject(this)
 
-    // Plan:
-    // - Show 'X' icon when a field is focused.
-    // - Switch between 'date of birth' and 'date of birth (dd/mm/yyyy)` for DOB's hint.
-
     fullNameEditText.showKeyboard()
     upButton.setOnClickListener { screenRouter.pop() }
```

Comment:
This should also be Back?

Reply:
This is not a part of this PR. I've so far used `up` everywhere else. I'll change them later.